### PR TITLE
Broke up the output further in line with changes on emission/core/wra…

### PR DESCRIPTION
**Pull request by trevor-wu**
…pper/suggestion_sys.py
- Broke up messages in accordance with changes on server, message now more detailed and controlled from clientside:
Testing Done:
Client w/o Suggestion (error message)
Screen Shot 2019-04-20 at 2 09 23 AM
Screen Shot 2019-04-20 at 2 09 30 AM
On Diary Page:
Screen Shot 2019-04-20 at 2 15 57 AM
With Hard-coded test suggestion:
Screen Shot 2019-04-20 at 2 26 57 AM
Reviews:
Screen Shot 2019-04-20 at 2 30 25 AM
On Diary:
Screen Shot 2019-04-20 at 2 33 54 AM
Reviews:
Screen Shot 2019-04-20 at 2 34 06 AM

:

### CODE REVIEW

File: www/templates/diary/detail.html [hunk 1]
```diff
@@ -67,15 +62,24 @@
                 ng-click="refreshMap()">Refresh</button>
             <button class="button button-stable button-block"
                 ng-click="getIndividualSuggestion()">Get Suggestion</button>
+<<<<<<< HEAD
             <button class="button button-stable button-block"
                 ng-click="clickReview()">Get Reviews</button>
+=======
```
Comment (shankari) [inline www/templates/diary/detail.html]: You have an unresolved merge conflict here. I am surprised that this worked. Did you test after resolving the merge conflict?

File: www/templates/diary/detail.html [hunk 2]
```diff
@@ -85,5 +89,7 @@ <h4>Reviews<br /></h4>
           </li>
            </div>
               <button class="button button-block button-balanced button-outline back" ng-click="clickX()">Back</button>
+=======
```
Comment (shankari) [inline www/templates/diary/detail.html]: Ditto

File: www/js/recommendations.js [hunk 3]
```diff
@@ -47,10 +50,13 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
...
         console.log(result);
-        $scope.name = result.message;
-        $scope.mode = result.method;
+        $scope.message = result.message;
+        $scope.question = result.question;
+        $scope.loc = result.suggested_loc;
+        $scope.mode = "Also try " + result.method + " instead";
         $scope.bid = result.businessid;
         $scope.stars = result.rating;
+        $scope.rating = "img/small/small_"+$scope.stars+".png";
```
Comment (shankari) [inline www/js/recommendations.js]: Isn't this part of #561 ? Why is it showing up here?

**Comment by shankari** [comment]
Also, this is still pretty wordy. @valin has a concrete suggestion for making it less so in 
https://github.com/e-mission/e-mission-server/pull/655

File: www/js/recommendations.js (hunk 3 above)
Comment (trevor-wu) [reply www/js/recommendations.js]: this is implementing the same change on recommendations.js, it was only implemented on details.js before

**Comment by shankari** [comment]
@trevor-wu there is still a merge conflict (unresolved) in `detail.html`
You may want to pull and resolve.

Screen Shot 2019-04-20 at 6 55 13 PM

**Comment by shankari** [comment]
also, from https://github.com/e-mission/e-mission-phone/pull/562#discussion_r277135573
> Did you test after resolving the merge conflict?

You have to really test after every change. If a merge applies cleanly, you could make an exception but you definitely have to retest if you actually had to edit to resolve anything. In particular, I expect you to test after you resolve this pending merge conflict as well

**Comment by trevor-wu** [comment]
Screen Shot 2019-04-21 at 4 48 17 PM
Testing for merge conflict

**Comment by trevor-wu** [comment]
Screen Shot 2019-04-21 at 4 51 07 PM
Testing for merge conflict

**Comment by shankari** [comment]
@trevor-wu as I pointed out a day ago https://github.com/e-mission/e-mission-phone/pull/562#issuecomment-485217631 this is still pretty wordy and in fact, I think that it is wordier than the previous version (I don't think that the previous version has the text "How about this location?")

Can you indicate your understanding of the problem you are looking to solve and how this solves it?

Also, the version in the detail view

has two copies of the "Why not check out" sentence.

**Comment by trevor-wu** [comment]
I was trying to match the output of the server form e-mission/e-mission-server#655 .
Screen Shot 2019-04-22 at 3 00 50 AM
And the double printing has been fixed

**Comment by shankari** [comment]
I still don't like the wording (e.g. "through vehicle") but it *is* consistent with https://github.com/e-mission/e-mission-server/pull/655 so I am going to go ahead and merge

**Comment by shankari** [comment]
@trevorwu, where is the commit for your last change? I don't see anything after "Merge branch 'greentrip' into gtui-patches4"

**Comment by trevor-wu** [comment]
I didn't change the code when I tested again. "Merge branch 'greentrip' into gtui-patches4" was to update gtui-patches4 so that it includes the changes from the last pull request - namely separating the buttons. The code in the repo is the code I tested with, I ran pull before I ran the last simulation.

**Comment by shankari** [comment]
So if you didn't change anything, then why did the initial testing have the more verbose text?

**Comment by trevor-wu** [comment]
When I resolved the merge conflict (commit 6df70cd ) , I went and fixed the bug with the double output as well ("Why not...") appearing twice. Should have clarified that in the commit message. I also used a different hard coded suggestion, one more in line with #655

**Comment by trevor-wu** [comment]
Tldr, the verbose double output was fixed in 6df70cd

**Comment by shankari** [comment]
Deployed, should be live in ~ 10 mins.
